main-aws.tf

```diff
@@ -144,6 +170,32 @@ resource "aws_s3_bucket" "financials" {
     yor_trace            = "0e012640-b597-4e5d-9378-d4b584aea913"
   })
 
+
+  server_side_encryption_configuration {
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure bucket policy is enforced with least privileges for all AWS S3 buckets**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
Make sure to note Amazon S3 bucket access control lists (ACLs) that provide read, write, or full-access to 'Everyone'. It is recommended to provide access to S3 buckets to only authenticated users instead.

source: tenablecs-